prime-router/src/main/kotlin/SubmissionReceiver.kt

```diff
@@ -267,13 +275,18 @@ class UniversalPipelineReceiver : SubmissionReceiver {
         // check that our input is valid HL7. Additional validation will happen at a later step
 
         val report: Report
+        var originalReportFormat: Format = Report.Format.INTERNAL
 
         when (sender.format) {
             Sender.Format.HL7 -> {
                 val messages = HL7Reader(actionLogs).getMessages(content)
                 val isBatch = HL7Reader(actionLogs).isBatch(content, messages.size)
                 // create a Report for this incoming HL7 message to use for tracking in the database
 
+                if (!isBatch) {
```

Comment:
If it is not batched, why do we explicitly need to set it to Report.Format.HL7? These types of changes should have a comment to explain the why when the why is not clear.